app/views/user/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>List of all users:</h1>
```

Comment:
- [ ] When I tried to visit users index page the error in the image below was raised, it happened because you wrongly named the views file that contains the `users#index` and `users#show` pages. It should be `users` and you named it `user`. Please fix that.

![image](https://github.com/Uzair-Manzoor/blog-app/assets/68796232/ab15e837-8379-4ef6-93cc-d092b4b4899a)

![image](https://github.com/Uzair-Manzoor/blog-app/assets/68796232/1311cc9e-ea14-4b4a-a466-d3a181a8e7d3)

Reply:
Hi @Omar-Muhamad, thank you for the insightful feedback. the highlighted issue has been resolved as suggested.